### ISSUE TITLE
Many keyboard-focusable elements lack focus styling

### ISSUE THREAD

**Issue by ryanhofdotgov** [18F/e-QIP-prototype#332]
From e-QIP-prototype-truetandem created by [toolness](https://github.com/toolness) : ryanhofdotgov/e-QIP-prototype-truetandem#1334  

I noticed that a number of keyboard-focusable elements--mostly links--don't have any styling to indicate that they currently have keyboard focus.

These include, but are not limited to:

* The "Logout" link at the top-right of every page
* All links on the nav sidebar on the left side of every page
* The "close info block" links in help sections
* The "DSN" and "International number" links in contact information areas

This makes it difficult for keyboard users--who may be using the keyboard because they e.g. have an injury or disability that prevents them from making precise movements with a mouse--to use the site. You can learn more about this kind of thing at [outlinenone.com](http://www.outlinenone.com/).

If you don't like the idea of making the visual experience less aesthetically pleasing for mouse or touch-based users, there's a very new CSS pseudo-class called `:focus-ring` that can help with this.  It's not supported by many (possibly *any*) browsers yet but there are polyfills available; you can learn more in Rob Dodson's [Focus Ring! -- A11ycasts #16](https://www.youtube.com/watch?v=ilj2P5-5CjI&index=3&list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g).

**Comment by bpdesigns** [comment]
looks resolved
